packages/stark-ui/src/modules/dropdown/components/dropdown.component.spec.ts

```diff
@@ -561,6 +561,46 @@ describe("DropdownComponent", () => {
 				});
 			});
 
+			describe("Changing emitEvent properties ", () => {
+				let mockObserver: SpyObj<Observer<any>>;
+
+				it("When the formControl is disabled or enabled with emitEvent set to false the dropdown shouldn't trigger a 'valueChange' event", () => {
+					hostComponent.formControl.valueChanges.subscribe(mockObserver);
+
+					hostComponent.formControl.disable({ emitEvent: false });
+					hostFixture.detectChanges();
+
+					expect(component.disabled).toBe(true);
+
+					hostComponent.formControl.enable({ emitEvent: false });
+					hostFixture.detectChanges();
+
+					expect(component.disabled).toBe(false);
+					expect(mockObserver.next).not.toHaveBeenCalled(); // because the 'emitEvent' is false
+					expect(mockObserver.error).not.toHaveBeenCalled();
+					expect(mockObserver.complete).not.toHaveBeenCalled();
+				});
+
+				it("When the formControl is disabled or enabled with emitEvent with emitEvent set to true the dropdown shouldn't trigger a 'valueChange' event", () => {
+					hostComponent.formControl.valueChanges.subscribe(mockObserver);
+
+					hostComponent.formControl.disable(); // 'emitEvent' true by default
+					hostFixture.detectChanges();
+
+					expect(component.disabled).toBe(true);
+					expect(mockObserver.next).toHaveBeenCalledTimes(1);
```

Comment:
Could you please add those 2 checks ?
```typescript
expect(mockObserver.error).not.toHaveBeenCalled();
expect(mockObserver.complete).not.toHaveBeenCalled();
```

Otherwise, if they are called after (line 599), you cannot be sure they were not called before...

Reply:
added